Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 gem 'github-pages', '>=39'
 gem 'jekyll-sitemap'
+
+gem "webrick", "~> 1.7"
```

Comment:
What is webrick used for?

Reply:
webrick was a required dependency for me to run this locally, it can be removed. good catch.